etna/pipeline/base.py

```diff
@@ -569,7 +579,7 @@ def _make_predict_timestamps(
         if end_timestamp > max_timestamp:
             raise ValueError("Value of end_timestamp is more than ending of dataset!")
 
-        if start_timestamp > end_timestamp:
+        if start_timestamp > end_timestamp:  # type: ignore
```

Comment:
Why do we get type problem here?

Reply:
It fails to understand that `start_timestamp` and `end_timestamp` are the same type here because we checked it by `_check_timestamp_param` higher up.